apps/model_eval_viz/app.py

```diff
@@ -1,41 +1,105 @@
-dfi_sine = hvplot.bind(sine, freq, phase).interactive()
-dfi_cosine = hvplot.bind(cosine, freq, phase).interactive()
+# Read in the dataframe 
+def _get_data(_filepath:str)->gpd.GeoDataFrame:
+    '''
+    Reads streamflow data from a .csv and filters it based on the 'gagesII_class==ref'.
+    Args:
+        _filepath (str): Path to the .csv file 
+    Returns:
+        gpd.GeoDataFrame: the filtered geopandas data file
+    '''
+    read_data = gpd.read_file(_filepath)
+    filtered_data = read_data[read_data['gagesII_class'] == 'Ref']
+    # clean up long, and lat data
+    filtered_data['dec_lat_va'] = filtered_data['dec_lat_va'].str.replace('lat', '').astype(float)
+    filtered_data['dec_long_va'] = filtered_data['dec_long_va'].str.replace('long', '').astype(float)
+    return filtered_data
+
+# Define data frames 
+gv_us_map = gv.Polygons(gv_us)
 
 # Plotting configurations
 plot_opts = dict(
-    responsive=True, min_height=400,
-    # Align the curves' color with the template's color
-    color=pn.template.FastGridTemplate.accent_base_color
+    #Dimensions, and UI setup
+    responsive=True, projection = ccrs.PlateCarree(), width=800, height=600, xlim=(-125, -50), ylim=(5, 50),
+    #title
+    title='United States Streamgage Map'
 )
 
-# Instantiate the template with widgets displayed in the sidebar
-template = pn.template.FastGridTemplate(
-    title="HyTEST Model Evaluation",
-    sidebar=[freq, phase],
-    
+# Instantiate template
+model_eval = pn.template.FastGridTemplate(
+    title="HyTEST Model Evaluation",  
 )
-footer = pn.pane.Markdown("""For questions about this application, please visit the [Hytest Repo](https://github.com/hytest-org/hytest/issues)""" ,width=500)
 
+# Plotting and Servable execution 
+stream_gage = _get_data(path)
+# Features = gv.Overlay([gf.ocean, gf.land, gf.rivers, gf.lakes, gf.borders, gf.coastline])
+features = gv.Polygons(states, crs=mapproj)
+
+# Widget setup to select multiple states
+state_selector = pn.widgets.MultiSelect(
+    description="Hold ctrl to toggle multiple states",
+    name="Select a state",
+    options=[
+
+"AL   Alabama", "AZ   Arizona", "AR   Arkansas", "CA   California", "CO   Colorado", "CT   Connecticut", "DE   Delaware", "FL   Florida", "GA   Georgia", 
+"ID   Idaho", "IL   Illinois", "IN   Indiana", "IA   Iowa", "KS   Kansas", "KY   Kentucky", "LA   Louisiana", "ME   Maine", "MD   Maryland", "MA   Massachusetts", "MI   Michigan", "MN   Minnesota", "MS   Mississippi", "MO   Missouri", "MT   Montana", "NE   Nebraska", "NV   Nevada", "NH   New Hampshire", "NJ   New Jersey", "NM   New Mexico", "NY   New York", "NC   North Carolina", "ND   North Dakota", "OH   Ohio", "OK   Oklahoma", "OR   Oregon", "PA   Pennsylvania", "RI   Rhode Island", "SC   South Carolina", "SD   South Dakota", "TN   Tennessee", "TX   Texas", "UT   Utah", "VT   Vermont", "VA   Virginia", "WA   Washington", "WV   West Virginia", "WI   Wisconsin", "WY   Wyoming"],
+)
```

Comment:
For these options, this seems like a lot to have in one line. Lets break this out into a config.py file. This will be used for constants in the app and other config that is needed. For example:

config.py
```python
STATES = ["AL   Alabama", "AZ   Arizona", "AR   Arkansas", "CA   California", "CO   Colorado", "CT   Connecticut", "DE   Delaware", "FL   Florida", "GA   Georgia", 
"ID   Idaho", "IL   Illinois", "IN   Indiana", "IA   Iowa", "KS   Kansas", "KY   Kentucky", "LA   Louisiana", "ME   Maine", "MD   Maryland", "MA   Massachusetts", "MI   Michigan", "MN   Minnesota", "MS   Mississippi", "MO   Missouri", "MT   Montana", "NE   Nebraska", "NV   Nevada", "NH   New Hampshire", "NJ   New Jersey", "NM   New Mexico", "NY   New York", "NC   North Carolina", "ND   North Dakota", "OH   Ohio", "OK   Oklahoma", "OR   Oregon", "PA   Pennsylvania", "RI   Rhode Island", "SC   South Carolina", "SD   South Dakota", "TN   Tennessee", "TX   Texas", "UT   Utah", "VT   Vermont", "VA   Virginia", "WA   Washington", "WV   West Virginia", "WI   Wisconsin", "WY   Wyoming"]
```

app.py
```python
# other imports
from config import *

###rest of app
    name="Select a State", 
    options=STATES)

Reply:
Another import option 
```python
import config

###later in code
    options=config.STATES
```

Reply:
We are already given the state names with the data set we are using so in future versions, I used 
state_list = states['shapeName'].unique()
Since our state boundaries already has all of the state names already.

Reply:
Great catch and a good way to implement this.